formcentral/config.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
I like the creation of a config.go file, this pattern should be followed everywhere!